### PR TITLE
[3.6] - "Revert bpo-24746: Avoid stripping trailing whitespace in doctest fancy diff (GH-10639)"

### PR DESCRIPTION
**Pull request by orsenthil**
Reverts python/cpython#11477

3.6 is in security fixes only mode and the patch merge was a mistake.

https://bugs.python.org/issue24746

### CODE REVIEW

**Comment by vstinner** [comment]
The commit message is confusing. I read it as the fix, whereas it's a revert.

**Review by vstinner — APPROVED** [review]
LGTM.

@ned-deily would you mind to merge this change?

**Comment by orsenthil** [comment]
@vstinner : I changed the title. I just need to "[3.6]" to satisfy the the status check, but I removed the "Revert" and it caused some confusion.

Should we wait for @ned-deily  or shall we  merge this once the CI goes green? I seem to have the privs to merge this PR.

**Comment by vstinner** [comment]
> Should we wait for @ned-deily or shall we merge this once the CI goes green? I seem to have the privs to merge this PR.

I'm not allowed to merge PRs in the 3.6 branch, the branch is restricted to release managers like @ned-deily.

**Pull request merged**
